lingua_franca/res/countries.json

```diff
@@ -0,0 +1,3227 @@
+[
+    {
+        "ISO3166-1-Alpha-3": "TWN",
+        "ISO3166-1-Alpha-2": "TW",
+        "official_name_fr": "",
+        "ISO4217-currency_name": "",
+        "ISO4217-currency_alphabetic_code": "",
+        "official_name_ar": "",
+        "official_name_es": "",
+        "official_name_cn": "",
+        "official_name_en": "",
+        "official_name_ru": "",
+        "Language": "zh-TW"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AFG",
+        "ISO3166-1-Alpha-2": "AF",
+        "official_name_fr": "Afghanistan",
+        "ISO4217-currency_name": "Afghani",
+        "ISO4217-currency_alphabetic_code": "AFN",
+        "official_name_ar": "أفغانستان",
+        "official_name_es": "Afganistán",
+        "official_name_cn": "阿富汗",
+        "official_name_en": "Afghanistan",
+        "official_name_ru": "Афганистан",
+        "Language": "fa-AF"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ALB",
+        "ISO3166-1-Alpha-2": "AL",
+        "official_name_fr": "Albanie",
+        "ISO4217-currency_name": "Lek",
+        "ISO4217-currency_alphabetic_code": "ALL",
+        "official_name_ar": "ألبانيا",
+        "official_name_es": "Albania",
+        "official_name_cn": "阿尔巴尼亚",
+        "official_name_en": "Albania",
+        "official_name_ru": "Албания",
+        "Language": "sq"
+    },
+    {
+        "ISO3166-1-Alpha-3": "DZA",
+        "ISO3166-1-Alpha-2": "DZ",
+        "official_name_fr": "Algérie",
+        "ISO4217-currency_name": "Algerian Dinar",
+        "ISO4217-currency_alphabetic_code": "DZD",
+        "official_name_ar": "الجزائر",
+        "official_name_es": "Argelia",
+        "official_name_cn": "阿尔及利亚",
+        "official_name_en": "Algeria",
+        "official_name_ru": "Алжир",
+        "Language": "ar-DZ"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ASM",
+        "ISO3166-1-Alpha-2": "AS",
+        "official_name_fr": "Samoa américaines",
+        "ISO4217-currency_name": "US Dollar",
+        "ISO4217-currency_alphabetic_code": "USD",
+        "official_name_ar": "ساموا الأمريكية",
+        "official_name_es": "Samoa Americana",
+        "official_name_cn": "美属萨摩亚",
+        "official_name_en": "American Samoa",
+        "official_name_ru": "Американское Самоа",
+        "Language": "en-AS"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AND",
+        "ISO3166-1-Alpha-2": "AD",
+        "official_name_fr": "Andorre",
+        "ISO4217-currency_name": "Euro",
+        "ISO4217-currency_alphabetic_code": "EUR",
+        "official_name_ar": "أندورا",
+        "official_name_es": "Andorra",
+        "official_name_cn": "安道尔",
+        "official_name_en": "Andorra",
+        "official_name_ru": "Андорра",
+        "Language": "ca"
```

Comment:
`ca-AD`?

Reply:
Not sure if this is accurate and it would be impossible to validate. We can stick with "official" dataset unless/until someone who knows corrects it.